erpnext/support/doctype/issue/issue.js

```diff
@@ -1,6 +1,13 @@
 frappe.ui.form.on("Issue", {
 	onload: function(frm) {
 		frm.email_field = "raised_by";
+		frm.set_query('customer', function () {
```

Comment:
```suggestion
		frm.set_query("customer", function () {
```
sorry but consistency